core/src/main/java/com/novoda/noplayer/AdvertsLoader.java

```diff
@@ -0,0 +1,15 @@
+package com.novoda.noplayer;
+
+import java.util.List;
+
+public interface AdvertsLoader {
```

Comment:
This is the basic API for ads loading. This let's the client control how they want to handle the loading in their own terms.